doc/source/installation.rst

```diff
@@ -218,11 +218,11 @@ path.
 
       3. make sure the NVCC compiler is available from ``CUDA_PATH/bin/nvcc``, otherwise the compilation may fail. You can locate it with ``whereis nvcc`` and eventually link/copy to your ``CUDA_PATH/bin`` folder.
 
-      For example, TensorFlow 2.4 supports CUDA 11. After installing
-      TensorFlow proceed with the NVCC 11 installation. On linux the
-      installation path usually is ``/usr/local/cuda-11/``.
+      For example, TensorFlow 2.5.0 supports CUDA 11.2. After installing
+      TensorFlow proceed with the NVCC 11.2 installation. On linux the
+      installation path usually is ``/usr/local/cuda-11.2/``.
```

Comment:
For me this path is just `/usr/local/cuda`. I am not sure if there is a standard, I guess it depends on how people install CUDA.

Reply:
Yeah, depends on how you install the package, e.g. ubuntu deb or directly from nvidia.